docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -86,6 +86,55 @@ to discuss reasonable assumptions with subject matter experts to determine the
 most appropriate measure to which to apply the GBD risk effects in our vivarium
 simulations.
 
+In GBD 2021, some continuous risk exposures were modeled with general
+relationships between the exposure level and the relative risk (going
+beyond the log-linear relationship assumed for previous iterations).
+Interpreting the GBD estimates is straightforward, once you have
+chased down all of the necessary definitions.  The relevant estimates
+include a column for exposure level, as well as columns for 500 draws
+of relative risk values at each exposure level.  The GBD 2021 PAF
+calculator often selected a TMREL for each draw from a uniform
+distribution, but for some risk factors, analysts provided draws for
+the TMREL as well.  The precise calculation to go from exposure levels
+and GBD-recorded risks to a function suitable for use as
+:math:`f_{rr}` as defined below are perhaps most clearly represented
+as python code:
+
+.. code-block:: python
+
+    def get_rr(draw_id):
+        # select rows for this sex and age_group
+        t1 = df[
+                (df.rei_id == rei_id)
+                & (df.cause_id == cause_id)
+                & (df.age_group_id == age_group_id)
+                & (df.sex_id == sex_id)]
+
+        # select column for requested draw
+        # and use exposure as index
+        t2 = t1.set_index('exposure').sort_index()
+        t3 = t2[f'draw_{draw_id}']
+
+        # make a rr function for this (risk, outcome, age, sex)
+        # by linear interpolation
+        import scipy.interpolate
+        x = np.array(t3.index) # df.exposure
+        y = np.array(t3.values)  # df.draw_n
+        f_rr = scipy.interpolate.interp1d(x, y, kind='linear', bounds_error=False, fill_value=(min(y), max(y)))
+
+        # pick a tmrel between tmred.min and tmred.max --- for certain risk factors, the modeling team uploads a model for this, too --- shiny tool has information about this
+        # and calculate relative risk at tmrel
+        tmrel = np.random.uniform(tmred.min, tmred.max)
+        rr_at_tmrel = f_rr(tmrel)
+        t4 = t3 / rr_at_tmrel
```

Comment:
FWIW, it's not at all obvious to me that this rescaling is necessary for our purposes. I think if you didn't do it (and clipped to `rr_at_tmrel` instead of 1), you'd get a different PAF from GBD, but the derived quantities we actually use ($i$ in the equations below) would be the same. Does that seem right to you?

I see why we might prefer to still do it, to have PAFs compatible with GBD, but I'd like to note it somewhere. And when we are calculating our own PAFs it just adds an extra step that may be confusing. I think this is a broader conversation that we should continue beyond this PR.

Reply:
That is fair. I think this is a longer-term thing to improve.

Basically, I'd like that if someone isn't using GBD data, they don't need to have a PAF at all -- it is an extra step that is only introduced by the fact that GBD doesn't give us exactly what we need for simulation.

Reply:
My thinking is that there will always be a need for "risk deleting" the rate to be modified --- we might as well call it a PAF.

Reply:
But for it to _really_ be a PAF we need to have a TMREL, which might not be relevant to our simulation's research question.